mujinvisioncontrollerclient/visioncontrollerclient.py

```diff
@@ -311,52 +311,75 @@ def StopTask(self, taskId=None, taskIds=None, taskType=None, taskTypes=None, cyc
         """Stops a set of tasks that meet the filter criteria
 
         Args:
-            taskId (optional): if specified, the specific taskId to stop
-            taskType (optional): if specified, only stop tasks of this task type
-            taskTypes (optional): if specified, a list of task types to stop
-            waitForStop (optional): if True, then wait for task to stop, otherwise just trigger it to stop, but do not wait
-            removeTask (optional): if True, then remove the task from being tracked by the vision manager and destroy all its resources. Will wait for the task to end before returning.
+            taskId (str, optional): If specified, the specific taskId to stop
+            taskIds (list[str], optional): If specified, a list of taskIds to stop
```

Comment:
Why do we have `taskId` and `taskIds`?
(same for `taskType(s)`)

You want to avoid changing the api of the client?

Reply:
I'm just documenting what's there.

And yes, changing the API is out of scope for this PR